husky-communication/husky-communication-gen/src/main/java/org/projecthusky/communication/configuration/HuskyConfiguration.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * 
+ */
+package org.projecthusky.communication.configuration;
+
+import org.apache.cxf.transports.http.configuration.HTTPClientPolicy;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Configuration clas for specific husky beans and other configs.
+ */
+@Configuration
+public class HuskyConfiguration {
+
+	@Value("${husky.communication.timeout:0}")
```

Comment:
Default value should correspond to the default behaviour needed for the EPR, i.e. 5 or 300 or 1800 (depending on the unit used)

Reply:
Just checked the specification and it is milliseconds, so default needs to be 1800. Having a default value of 0 will even overwrite the default value used by HttpClientPolicy

Reply:
Why these values? Is there anywhere in the EPR specifications where socket timeouts (connection/read) values are discussed?

1.8 seconds is particularly dubious, but I agree that we should not have an infinite value by default.

Reply:
Timeout in EPR is minimum 60 seconds.

Reply:
Sorry, @qligier my mistake, factor s to ms is 1000 not 60 :) - @ralych oversaw that 0 is infinite, fine for me.